bigbluebutton-html5/imports/ui/components/whiteboard/service.js

```diff
@@ -39,6 +49,9 @@ function handleAddedAnnotation({
       $inc: { version: 1 }, // TODO: Remove all this version stuff
     });
 
+    if (annotation.status === 'DRAW_END') {
```

Comment:
You can use the whiteboard setting in setting.json.

Reply:
You mean getting "annotation.satus" from setting.json?

Reply:
Yes